addons/account_edi_ubl_cii/models/account_edi_common.py

```diff
@@ -670,6 +679,8 @@ def _retrieve_line_vals(self, tree, document_type=False, qty_factor=1):
             'discount': discount,
             'tax_nodes': self._get_tax_nodes(tree),  # see `_retrieve_taxes`
             'charges': charges,  # see `_retrieve_line_charges`
+            'deferred_start_date': start_date,
```

Comment:
This returns is not taking care of the previous check, creating an error in single modules and community builds.